### DISCUSSION TITLE
Manual refresh and automatic refresh occasionally refresh the articles that have been marked as read!

### DISCUSSION THREAD

**Discussion by ghost** [FreshRSS/FreshRSS#5123] [Q&A] [answered]
Hello, developer, I use FreshRSS. The subscribed content is entirely in Chinese. Search the search engine Google and Bing for "refresh the articles you have read" to find FreshRSS users with the same problems as me.
The problem is that articles that have been marked as read often appear after I refresh the feed manually or regularly.
I sincerely hope that developers can test this problem and fix it! Thank you. I'm sorry to bother you. This problem has a great impact on your use. thank you!

**Comment by Frenzie** [comment, marked as answer]
There are generally two reasons something might be marked as unread.

Presumably you already checked, but there's Settings → Reading → Mark updated articles as unread. I believe that's disabled by default, but perhaps it was enabled.

The second reason can't really be helped. That's if the same text is published with a different guid/permalink. A sample feed or sample feed content from two relevant points in time would be needed to say more.

**Comment by ghost** [comment]
After closing the "updated article as read" two days ago, there has been no problem. Thank you, good evening.

**Comment by aledeg** [comment]
Can we close this issue then?
